skimage/feature/tests/test_corner.py

```diff
@@ -351,17 +351,22 @@ def test_num_peaks():
 def test_corner_peaks():
     response = np.zeros((10, 10))
     response[2:5, 2:5] = 1
+    response[8:10, 0:2] = 1
 
     corners = corner_peaks(response, exclude_border=False, min_distance=10,
                            threshold_rel=0)
     assert len(corners) == 1
 
+    corners = corner_peaks(response, exclude_border=False, min_distance=5,
+                           threshold_rel=0)
+    assert len(corners) == 2
+
     corners = corner_peaks(response, exclude_border=False, min_distance=1)
-    assert len(corners) == 4
+    assert len(corners) == 5
```

Comment:
Because you're already touching this test, would you mind asserting the full shape instead of only the first dimension via the `len()` function? E.g.

```
assert corners.shape == (5, 2)
```
should be more readable.

Reply:
@lagru Thank you! No problem at all, that change makes sense to me. Just pushed your revision up.